gestalt-module/src/main/java/org/terasology/gestalt/module/sandbox/APIScanner.java

```diff
@@ -34,10 +34,20 @@ public class APIScanner {
     private final StandardPermissionProviderFactory permissionProviderFactory;
     private final ClassLoader forClassLoader;
 
+    /**
+     * Sets a permissionProviderFactory to scan a reflections manifest for API annotated classes using the system classloader.
+     * @param permissionProviderFactory interface for factories that produce permission providers for modules.
```

Comment:
```suggestion
     * Creates an APIScanner instance to scan reflection manifests for API annotated classes using the system classloader.
     * @param permissionProviderFactory a factory that provides permissions granted to discovered modules.
```

I am sorry to say that in its current form this does not make much sense? This is also a constructor, not a setter.